File: Team1/RiyaSingh/removeDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n = nums.length;
        if(n==0 || n==1){
            return n;
        }
        int temp[] = new int[n];
        int j = 0;
        for(int i=0;i<n-1;i++){
            if(nums[i]!=nums[i+1]){
                temp[j++] = nums[i];
            }
        }
        temp[j++] = nums[n-1];
        for(int i = 0;i<j;i++){
            nums[i] = temp[i];
        }
        return j;
    }
}

File: Team5/Akanksha_Singh/Week-1/RemoveDuplicates.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int temp=1,j=1;
        for(int i=1;i<nums.size();i++)
        {
            if(nums[i]!=nums[i-1])
            {
                temp++;
                nums[j]=nums[i];
                j++;
            }
        }
        return temp;
    }
};
